.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,12 @@
+name: Scheduled
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
```

Comment:
I think we need to add above this:

```yaml
permissions:
   issues: write
```

docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions